src/spec/connect/rest.spec.coffee

```diff
@@ -3,6 +3,13 @@ _.mixin require('underscore-mixins')
 {Rest} = require '../../lib/main'
 Config = require('../../config').config
 
+idCounter = 0
+uniqueId = (prefix) ->
+  id = ++idCounter + ''
+  if prefix then prefix + id else id
+resetUniqueIdCounter = () ->
+  idCounter = 0
```

Comment:
We should add a note here why we need to have a custom implementation. Actually we can scope it within the PAGED test only.